test/e2e/setup/setup.go

```diff
@@ -160,52 +191,41 @@ func deployOperator(ctx *e2eutil.Context) error {
 	}
 	fmt.Printf("Setting namespace to watch to %s\n", watchNamespace)
 
-	if err := buildKubernetesResourceFromYamlFile(ctx, path.Join(roleDir(), "role.yaml"), &rbacv1.Role{}, withNamespace(testConfig.namespace)); err != nil {
-		return errors.Errorf("error building operator role: %s", err)
+	chartName := "mongodb-kubernetes-operator"
+	if err := helm.Uninstall(chartName); err != nil {
+		return err
 	}
-	fmt.Println("Successfully created the operator Role")
 
-	if err := buildKubernetesResourceFromYamlFile(ctx, path.Join(roleDir(), "service_account.yaml"), &corev1.ServiceAccount{}, withNamespace(testConfig.namespace)); err != nil {
-		return errors.Errorf("error building operator service account: %s", err)
+	helmChartPath := envvar.GetEnvOrDefault(helmChartPathEnv, "/workspace/helm-chart")
+	helmArgs := getHelmArgs(testConfig, watchNamespace)
+	if err := helm.Install(helmChartPath, chartName, helmArgs); err != nil {
+		return err
 	}
-	fmt.Println("Successfully created the operator Service Account")
 
-	if err := buildKubernetesResourceFromYamlFile(ctx, path.Join(roleDir(), "role_binding.yaml"), &rbacv1.RoleBinding{}, withNamespace(testConfig.namespace)); err != nil {
-		return errors.Errorf("error building operator role binding: %s", err)
+	dep, err := waite2e.ForDeploymentToExist("mongodb-kubernetes-operator", time.Second*10, time.Minute*1, e2eutil.OperatorNamespace)
+	if err != nil {
+		return err
 	}
-	fmt.Println("Successfully created the operator Role Binding")
 
-	if err := buildKubernetesResourceFromYamlFile(ctx, path.Join(roleDir(), "role_database.yaml"), &rbacv1.Role{}, withNamespace(testConfig.namespace)); err != nil {
-		return errors.Errorf("error building mongodb database role: %s", err)
-	}
-	if err := buildKubernetesResourceFromYamlFile(ctx, path.Join(roleDir(), "service_account_database.yaml"), &corev1.ServiceAccount{}, withNamespace(testConfig.namespace)); err != nil {
-		return errors.Errorf("error building mongodb database service account: %s", err)
+	quantityCPU, err := resource.ParseQuantity("50m")
+	if err != nil {
+		return err
 	}
-	if err := buildKubernetesResourceFromYamlFile(ctx, path.Join(roleDir(), "role_binding_database.yaml"), &rbacv1.RoleBinding{}, withNamespace(testConfig.namespace)); err != nil {
-		return errors.Errorf("error building mongodb database role binding: %s", err)
+
+	for _, cont := range dep.Spec.Template.Spec.Containers {
```

Comment:
reduce resources used by the operator as this can cause issues with the tests if they are too high.